lab_3/main.py

```diff
@@ -107,7 +108,14 @@ def predict_next_sentence(self, prefix: tuple) -> list:
 
 
 def encode(storage_instance, corpus) -> list:
-    pass
+    new_corpus = []
+    new_sentence = []
+    for sentence in corpus:
+        for word in sentence:
+            new_word = storage_instance[word]
```

Comment:
@pdpetrova storage_instance is not a dictionary here, you a supposed to pass the whole object WordStorage:
```python
ws = WordStorage()
...
encoded = encode(ws, sentences)
```